### PR TITLE
Reordered sections within user docs

### PR DESCRIPTION
**Pull request by ForgottenProgramme**
**The user documentation contains 7 subsections. The subsections are reordered, so that there is a better ‘flow of information’ and  the most frequently required information is at the top.** 

Fixes part of #1454 

userdoc1

### CODE REVIEW

**Comment by ForgottenProgramme** [comment]
New commit: 17/06/2021

Added 'Can I contribute packages to conda-forge?' section in user docs. 

contri

**Comment by ForgottenProgramme** [comment]
@viniciusdc

File: src/user/introduction.rst [hunk 1]
```diff
@@ -94,7 +94,15 @@ From now on using ``conda install <package-name>`` will also find packages in ou
   Please refer to :ref:`multiple_channels` for pitfalls and more information.
 
 
-How can I give credit to ``conda-forge``?
+Can I contribute packages to conda-forge?
```
Comment (viniciusdc) [inline src/user/introduction.rst:97]:
```suggestion
Becoming Involved: How to contribute packages to conda-forge?
```

File: src/user/introduction.rst [hunk 2]
```diff
@@ -94,7 +94,15 @@ From now on using ``conda install <package-name>`` will also find packages in ou
...
+-----------------------------------------
+
+Yes, anyone can contribute packages to the ``conda-forge`` channel.
```
Comment (viniciusdc) [inline src/user/introduction.rst:100]:
```suggestion
Anyone can contribute packages to the ``conda-forge`` channel. 
```

File: src/user/introduction.rst [hunk 3]
```diff
@@ -94,7 +94,15 @@ From now on using ``conda install <package-name>`` will also find packages in ou
   Please refer to :ref:`multiple_channels` for pitfalls and more information.
 
 
-How can I give credit to ``conda-forge``?
+Can I contribute packages to conda-forge?
+-----------------------------------------
+
+Yes, anyone can contribute packages to the ``conda-forge`` channel. 
+You don't have to be the upstream maintainer of a package in order to contribute it to ``conda-forge``. 
+To learn how you can contribute your first package read `the staging process <https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-staging-process>`_.
```
Comment (viniciusdc) [inline src/user/introduction.rst:102]:
```suggestion
If you are interested in submitting a package to conda-forge, read `the staging process <https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-staging-process>`_. for a better description of the process.
```
Maybe?

File: src/user/00_intro.rst [hunk 4]
```diff
@@ -5,9 +5,9 @@ User Documentation
 .. toctree::
 
   introduction
-  announcements
+  contributing
+  how_to_get_help
   tipsandtricks
   ci-skeleton
   faq
```
Comment (viniciusdc) [inline src/user/00_intro.rst:12]:
```suggestion
  announcements
  faq
```

**Review by viniciusdc — COMMENTED** [review]
Could you leave a comment explaining some of the benefits?

**Comment by ForgottenProgramme** [comment]
@viniciusdc 
Updated as suggested.

**Comment by viniciusdc** [comment]
This is part of reordering process and updating the user sections for the conda-forge documentation.

**Pull request closed without merging**